app/helpers/blacklight/configuration_helper_behavior.rb

```diff
@@ -87,13 +87,11 @@ def document_show_field_label document, field
   # Look up the label for the facet field
   def facet_field_label field
     field_config = blacklight_config.facet_fields[field]
+    defaults = [:"blacklight.search.fields.facet.#{field}", :"blacklight.search.fields.#{field}"]
+    defaults << field_config.label if field_config
+    defaults << field.to_s.humanize
 
-    field_label(
-      :"blacklight.search.fields.facet.#{field}",
-      :"blacklight.search.fields.#{field}",
-      (field_config.label if field_config),
-      field.to_s.humanize
-    )
+    field_label defaults
   end
```

Comment:
I don't think it matters. Somewhere in the stack, the `nil` values get compacted away.